### PR TITLE
fix 4.0 package.json

### PR DESCRIPTION
**Pull request by turret-dev**
update package name
move typescript types to devdependencies
fix homepage link
fix bugs link
fix repository link
fix version (3.2.0 to 4.0.0)
change main to out/index.js (since that is the actual main file)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
-  "name": "discordtel",
-  "version": "3.2.0",
+  "name": "dtel",
+  "version": "4.0.0",
   "description": "Discord Telephone",
-  "main": "index.js",
+  "main": "out/index.js",
```
Comment (Rexogamer) [inline package.json:5]: Shouldn't this be
```suggestion
  "main": "src/index.ts",
```

File: package.json [hunk 2]
```diff
@@ -20,17 +20,14 @@
...
+    "url": "https://github.com/dtel-hq/dtel/issues"
   },
-  "homepage": "https://discordtel.austinhuang.me",
+  "homepage": "https://dtel.austinhuang.me",
   "engines": {
     "node": "12"
   },
   "dependencies": {
     "@discoin/scambio": "^2.0.4",
-    "@types/rethinkdb": "^2.3.14",
```
Comment (Rexogamer) [inline package.json:31]: Are you sure these are devdeps? While these specific packages are types, RethinkDB/WS aren't listed as dependencies despite being required for running the bot (not just dev)

**Review by Rexogamer — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (SunburntRock89) [reply package.json:5]: no, it shouldn't, you don't enter at index.ts

File: package.json (hunk 2 above)
Comment (SunburntRock89) [reply package.json:31]: They're not dev deps

**Review by SunburntRock89 — CHANGES_REQUESTED** [review]
Move the devdeps back to main deps, this thing probably isn't gonna be used anyway because mitchell wants a rewrite

File: package.json (hunk 1 above)
Comment (Rexogamer) [reply package.json:5]: do we even compile the code? this isn't a published package after all

but either way there's no index.js and I've never seen out as a folder before (dist instead?)

Comment (turret-dev) [reply package.json:5]: index.ts is not a valid entry point

**Comment by turret-dev** [comment]
i mean you guys have node typings on dev dependencies, why not the others?

**Comment by SunburntRock89** [comment]
Node deps are included by default in TS

**Pull request closed without merging**